mapusaurus/api/views.py

```diff
@@ -32,11 +30,17 @@ def tables(request):
 def msas(request):
     """return a list of MSA ids visible by bounding coordinates"""
     try:
-        msas = get_censustract_geos(request, metro=True)
+        northEastLat = request.GET.get('neLat')
+        northEastLon = request.GET.get('neLon')
+        southWestLat = request.GET.get('swLat')
+        southWestLon = request.GET.get('swLon')
+        if northEastLat and northEastLon and southWestLat and southWestLon:
```

Comment:
Does .get() return None if the key doesn't exist?  I was under the impression that it'd throw an error...  I'd expect something like `northEastLat = request.GET.get('neLat', None)` to ensure a fallback default.

This would be a great function to have tests for, if they don't already exist, to verify functionality for all the combinations of neLat and/or neLon and/or swLat and/or swLon...